Demo/Demo/SampleViewController.swift

```diff
@@ -0,0 +1,72 @@
+import SwiftUI
+import UIKit
+import Components
+import WKData
+
+final public class SampleViewController: WKCanvasViewController {
```

Comment:
1. Looks like this accidentally got added to the root of the Demo project twice – can you remove this reference at the root?
<img width="336" alt="svc" src="https://github.com/wikimedia/wikipedia-ios-components/assets/5652327/d16f0db6-ad45-4066-887c-8ea5d75fb553">

2. Could we rename this (and related `Sample` classes here) to something more descriptive of the feature? Maybe `ProjectIconSample...` or something of the sort?

Reply:
Done!